diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -12,7 +12,7 @@ use sqlite::Sqlite;
 const SQLITE_DATE_FORMAT: &str = "%F";
 
 impl FromSql<Date, Sqlite> for NaiveDate {
-    fn from_sql(value: Option<&<Sqlite as Backend>::RawValue>) -> deserialize::Result<Self> {
+    fn from_sql<'a>(value: Option<<<Sqlite as Backend>::RawValue as FamilyLt<'a>>::Out>) -> deserialize::Result<Self> {
```

Comment:
Just use the actual RawValue type
```suggestion
    fn from_sql(value: Option<&SqliteValue>) -> deserialize::Result<Self> {
```